### ISSUE TITLE
Reconsider not publishing to Mitre ATT&CK

### ISSUE THREAD

**Issue by xg5-simon** [pushsecurity/saas-attacks#80]
This is a great collection of TTPs that I reference frequently but now that ATT&CK Enterprise has a [SaaS Matrix](https://attack.mitre.org/matrices/enterprise/cloud/saas/) is there any thought to submitting these techniques to ATT&CK?

This will help reach a wider audience and support downstream CTI pipelines via [MITRE's TAXII workbench](https://github.com/mitre-attack/attack-workbench-taxii-server).

**Comment by sudonoodle** [comment]
I agree with this. I also think this repository is more "identity" centric than "saas". 

At least, to me, "Identity attack matrix" makes more sense.
